src/components/SignerPanel/SignerPanel.js

```diff
@@ -123,21 +169,42 @@ class SignerPanel extends React.Component {
       onTransactionSuccess && onTransactionSuccess(transaction)
 
       transactionBag.accept(transactionRes)
-      this.setState({ signError: null, status: STATUS_SIGNED })
+      this.setState({ signError: null, status: STATUS_SIGNED_TX })
       this.startClosing()
 
       // Display an error in the panel if a transaction fail
     } catch (err) {
       transactionBag.reject(err)
-      this.setState({ signError: err, status: STATUS_ERROR })
+      this.setState({ signError: err, status: STATUS_ERROR_TX })
 
       // TODO: the ongoing notification should be flagged faulty at this point ...
     }
   }
 
+  handleMsgSign = async () => {
+    const { signatureBag, account } = this.props
+
+    this.setState({ status: STATUS_SIGNING_MESSAGE })
+    try {
+      const signatureHash = await this.signMessage(
```

Comment:
```suggestion
      const signature = await this.signMessage(
```

I think the return value is a signature. More specifically a concatenated r,s,v ECDSA tuple.